DEBUG    urllib3.connectionpool:connectionpool.py:232 Starting new HTTP connection (1): 127.0.0.1:6666
DEBUG    urllib3.connectionpool:connectionpool.py:465 http://127.0.0.1:6666 "POST /business/token_login HTTP/1.1" 200 130
ERROR    root:test_login05.py:75 测试编号6，测试标题测试用户名区分大小写的场景，执行失败，实际结果{'code': 1000, 'msg': '登录成功', 'token': 'MTY0Njg0NTE0NC42ODExOTg2OjExYmE4NmRkNjI5OWIzNGRhODc5MDU4OTQ1MTRmOTY4ZTcxY2E3ODU='}
ERROR    root:test_login05.py:76 assert {'code': 1007...: '用户名或者密码错误'} == {'code': 1000...ZTcxY2E3ODU='}
  Differing items:
  {'msg': '用户名或者密码错误'} != {'msg': '登录成功'}
  {'code': 1007} != {'code': 1000}
  Right contains 1 more item:
  {'token': 'MTY0Njg0NTE0NC42ODExOTg2OjExYmE4NmRkNjI5OWIzNGRhODc5MDU4OTQ1MTRmOTY4ZTcxY2E3ODU='}
  Full diff:
    {...
  
  ...Full output truncated (9 lines hidden), use '-vv' to show
Traceback (most recent call last):
  File "D:\PythonWorkSpace\autotest\testcases\test_business_token_login02\test_login05.py", line 72, in test_login
    assert eval(case.expect) == res
AssertionError: assert {'code': 1007...: '用户名或者密码错误'} == {'code': 1000...ZTcxY2E3ODU='}
  Differing items:
  {'msg': '用户名或者密码错误'} != {'msg': '登录成功'}
  {'code': 1007} != {'code': 1000}
  Right contains 1 more item:
  {'token': 'MTY0Njg0NTE0NC42ODExOTg2OjExYmE4NmRkNjI5OWIzNGRhODc5MDU4OTQ1MTRmOTY4ZTcxY2E3ODU='}
  Full diff:
    {...
  
  ...Full output truncated (9 lines hidden), use '-vv' to show